.github/workflows/dockertests.yml

```diff
@@ -120,6 +121,7 @@ jobs:
           echo "Test docker container image using a `.tar` file"
           docker pull elasticsearch@sha256:3686a5757ed46c9dbcf00f6f71fce48ffc5413b193a80d1c46a21e7aad4c53ad
           docker save -o /tmp/elastic.tar elasticsearch@sha256:3686a5757ed46c9dbcf00f6f71fce48ffc5413b193a80d1c46a21e7aad4c53ad
+          docker rmi elasticsearch@sha256:3686a5757ed46c9dbcf00f6f71fce48ffc5413b193a80d1c46a21e7aad4c53ad
```

Comment:
Are these changes to any for a different PR?

Reply:
Tests were failing for various reasons and this is one of them.